config/default/manager_credentials_patch.yaml

```diff
@@ -9,7 +9,7 @@ spec:
       containers:
       - name: manager
         env:
-        - name: GOOGLE_APPLICATION_CREDENTIALS
+        - name: AZURE_CREDENTIALS_FILE
```

Comment:
Is this correct for Azure? Do we need to pass the credentials in?

Reply:
Actually seems this has been removed in the other file

Reply:
which other file?

Reply:
`credentials.yaml`, the change shows that `credentials.json` is removed

Reply:
Yeah, we probably don't need this/I haven't used this before.
We're expecting env-based configuration.